recipes/diffusioncma/build.sh

```diff
@@ -0,0 +1,29 @@
+
+echo "Building libcmaes..."
+mkdir -p build_libcmaes
```

Comment:
Perhaps https://github.com/beniz/libcmaes should be a separate feedstock in itself? I am not against statically linking it here, but ... @xhochy might have a different viewpoint here.

Reply:
Well my viewpoint is that we don't have the tooling right now to correctly support static linkage. If it can be a separate feedstock (and it looks like in this case), we should make one and link to the shared library.

Reply:
That is unfortunate. Actually, [libcmaes already has a feedstock](https://anaconda.org/conda-forge/libcmaes), which works fine on OSX. Unfortunately, the tests were segfaulting for no apparent reason on Linux, see [linux pipeline](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=164858&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0) for commit 6d3b505. I thought that compiling the library during build and link statically would be a convenient way to work around this issue. (It is also quite undesirable that the libcmaes build on conda-forge does [not support py37](https://anaconda.org/conda-forge/libcmaes/files), see also [pipeline](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=164847&view=logs&jobId=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&j=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&t=81967b5f-0283-5c0b-87c1-e8fdeef842a3) for commit 7f5b8d3.) If I compile everything myself, I do not run into any of these issues. Compiling a shared library is probably not an option, since it could run into conflicts with the libcmaes feedstock, right? Is there anything else I could try?

Reply:
Please open an issue in `libcmaes-feedstock`. Vendoring packages like this is discouraged.

Reply:
The libcmaes feedstock looks abandoned and in dire need of updating. If the build here works with cmake, then it should also work on the feedstock. The libcmaes builds on conda-forge are on the old compiler toolchain and segfaults when using it with the new one are rather expected. 

I'm happy to review and merge PRs that fix that feedstock. This will probably be the easiest solution for all.

Reply:
The situation is as follows: the latest libcmaes [version 0.9.5](https://github.com/beniz/libcmaes/releases) is quite old (9 Mar 2015) and there have been many changes to the repo since then. Most importantly, the release version does not contain the cmake files that I am using for the build here. I [asked the developer](https://github.com/beniz/libcmaes/issues/205) (@beniz) if he would be willing to put out an updated release version, which would make things much easier for us.

Otherwise, I will have to go for an automake build in [the PR to the libcmaes feedstock](https://github.com/conda-forge/libcmaes-feedstock/pull/14) and see if the problem is due to the compiler toolchain (it is very possible that it is a problem with the libcmaes code though - given the amount of changes since the latest release).

If nothing else helps: is vendoring the package like I am doing now an option?

Reply:
Let's first try and update the feedstock. If vendoring actually works, updating the feedstock should also work.

Reply:
@xhochy Fixed. The libcmaes feedstock is updated and this one is building as well without vendoring.